python/llm/example/GPU/QLoRA-FineTuning/qlora_finetuning.py

```diff
@@ -41,11 +41,22 @@
 
     data = load_dataset(dataset_path)
     data = data.map(lambda samples: tokenizer(samples["quote"]), batched=True)
+
+    from transformers import BitsAndBytesConfig
+    nf4_config = BitsAndBytesConfig(
+        load_in_4bit=True,
+        bnb_4bit_quant_type="nf4",
+        bnb_4bit_compute_dtype=torch.bfloat16
```

Comment:
add bnb_4bit_use_double_quant=False
and add a warning message if it is set to True